Descripction: not using Google fonts anymore in order to avoid privacy breach.
Author: Marcos Fouces <marcos.fouces@gmail.com>
Forwarded: not-needed

--- a/flasgger/ui3/templates/flasgger/head.html
+++ b/flasgger/ui3/templates/flasgger/head.html
@@ -1,6 +1,5 @@
 <meta charset="UTF-8">
 <title>{{ title }}</title>
-<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet">
 <link rel="stylesheet" type="text/css" href="{{ swagger_ui_css }}">
 <!-- Customize the app.config['SWAGGER']['favicon'] -->
 <link rel="icon" type="image/png" href="{{ favicon }}" sizes="64x64 32x32 16x16" />
